skaffold.yaml

```diff
@@ -514,19 +514,19 @@ profiles:
   - command: debug
   patches:
   - op: add
-    path: /deploy/helm/releases/0/setValues
+    path: /deploy/helm/releases/0/setValueTemplates
     value:
       global.apiserver.replicaCount: 1
       global.apiserver.livenessProbe.enable: false
       global.apiserver.readinessProbe.enable: false
       global.admission.replicaCount: 1
-      global.admission.config.server.healthProbes.enable: false
+      global.admission.config.server.healthProbes.enable: "{{ default \"\" .SKAFFOLD_DISABLE_HEALTH_CHECKS | lower | splitList \",\" | has \"admission\" | ternary \"false\" \"true\" }}"
```

Comment:
Please see my first comment about `SKAFFOLD_DISABLE_HEALTH_CHECKS` env.

Reply:
May we merge this into the discussion above?